## Hi there 👋

My name is Clémence Frioux, welcome!

- 👩‍💻 I am a researcher in computational biology
- 📍 I work at Inria, a French research Institute on computer science and mathematics. I am located in Bordeaux, France
- 🧬 🦠 I am interested in the (functional) characterisation of microbial communities. I mostly work on metabolic modelling 🦠, but I also work on microbiome composition and metagenomics 🧬
- 😄 Pronouns: She/Her
- ➡️ More info on my website: [https://cfrioux.github.io]

## A few projects hosted here on Github that I want to highlight
I was not a developer in all of those projects but participated to them. 

### 🧬 Mapler

Evaluation of assembly and binning quality for Hi-Fi metagenomics.

- [Repo](https://github.com/Nimauric/Mapler#)
- [Paper](https://academic.oup.com/bioinformatics/article/41/6/btaf334/8157874)

### 🧬 cvaNMF

A package for non-negative factorisation and its bi-cross validation. 

- [Repo](https://github.com/apduncan/cvanmf)
<!-- [Paper]() -->

### 🦠 Metage2Metabo

Pipeline dedicated to the (optional) automatic reconstruction of genome-scale metabolic network and characterising the metabolic potential and metabolic complementarity in microbial communities. 

- [Repo](https://github.com/AuReMe/metage2metabo)
- [Paper](https://doi.org/10.7554/eLife.61968)

### 🦠 Metage2Metabo-postAViz

Shiny-based interface to integrate results of Metage2Metabo together with sample and taxonomic metadata, in order to compare metabolic potential across large cohorts of samples. 

- [Repo](https://github.com/AuReMe/metage2metabo-postaviz)


### 🦠 MeneTools

Several tools relying on network expansion to explore metabolic networks.

- [Repo](https://github.com/cfrioux/MeneTools)
- [Paper](https://doi.org/10.1371/journal.pcbi.1006146)

### 🦠 MiSCoTo

Selection of minimal communities ensuring the reachability of target metabolites, using a collection of metabolic networks

- [Repo](https://github.com/cfrioux/miscoto/tree/main)
- [Paper](https://academic.oup.com/bioinformatics/article/34/17/i934/5093211)

### 🦠 Seed2LP

Inference of seeds (environmental composition) from metabolic networks, using network expansion and flux balance analysis. Can be used for individual populations and microbial communities.

- [Repo](https://github.com/bioasp/seed2lp)
- [Paper for individual populations](https://academic.oup.com/bioinformatics/article/41/4/btaf140/)
- [Paper for microbial communities](https://link.springer.com/chapter/10.1007/978-3-032-01436-8_20)

### 🦠 Meneco

Gap-filling of metabolic networks.

- [Repo](https://github.com/cfrioux/meneco)
- [Paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005276)

### 🦠 AuCoMe

Automatic reconstruction and comparison of metabolic networks. 

- [Repo](https://github.com/AuReMe/aucome)
- [Paper](https://doi.org/10.1101/gr.277056.122)

### 🦠 mpwt

Python wrapper for Pathway Tools (reconstruction of metabolic networks).

- [Repo](https://github.com/AuReMe/mpwt/tree/e95165878dcaa422ef4cef96d75489fc7d0ad677)
- [Paper](https://doi.org/10.7554/eLife.61968)
